modules/svg/svg.go

```diff
@@ -4,10 +4,67 @@
 
 package svg
 
-// SVGs contains discovered SVGs
-var SVGs map[string]string
+import (
+	"fmt"
+	"html/template"
+	"regexp"
+	"strings"
+)
+
+var (
+	// SVGs contains discovered SVGs
+	SVGs map[string]string
+
+	widthRe  = regexp.MustCompile(`width="[0-9]+?"`)
+	heightRe = regexp.MustCompile(`height="[0-9]+?"`)
+)
 
 // Init discovers SVGs and populates the `SVGs` variable
 func Init() {
 	SVGs = Discover()
 }
+
+// ParseOthers get size and class from string with default values
+func ParseOthers(defaultSize int, defaultClass string, others ...interface{}) (int, string) {
```

Comment:
`ParseSizeAndClass`?

Reply:
Also, I'm not quite sure if the `svg` package is the correct place for this function, as it could also be used for other image types, see i.e. the `Avatar()` call below.

Reply:
The function parses shared attributes from Avatar and SVG template helpers. I'd mention that in the description. Name `ParseSizeAndClass` sounds fine. Maybe move it to maybe `module/util`?

Reply:
I agree `ParseSizeAndClass` should be a better name but move it to `module/util` is not better than keeping it here. `util` is not a good package name.

Reply:
Moved to `modules/html`.